content/en/datadog_cloudcraft/_index.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Cloudcraft in Datadog
+further_reading:
+- link: "https://www.datadoghq.com/blog/cloud-architecture-diagrams-cost-compliance-cloudcraft-datadog/"
+  tag: "Blog"
+  text: "Plan new architectures and track your cloud footprint with Cloudcraft (Standalone)"
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">Cloudcraft in Datadog is not supported on this site.</div>
+{{< /site-region >}}
+
+## Overview
+
+Cloudcraft offers a powerful, live read-only visualization tool for cloud architecture, enabling you to explore, analyze, and manage your infrastructure with ease. Not to be confused with the [Standalone Cloudcraft documentation][1], this guide outlines the functionality, setup, and use cases of Cloudcraft *in Datadog*, detailing its benefits for various user personas, and highlighting key features and capabilities.
+
+<div class="alert alert-info">This documentation applies to the Cloudcraft <em>in Datadog</em> product. For information on the standalone Cloudcraft product, please refer to the <a href="/cloudcraft"> Cloudcraft (Standalone) </a> documentation.</div>
+
+Cloudcraft's core functionality is its ability to generate detailed architecture diagrams. These diagrams visually represent AWS cloud resources, allowing you to explore and analyze your environments. Cloudcraft's diagrams are optimized for clarity and performance, providing an intuitive interface for navigating large-scale deployments. This helps teams to:
+
+- Trace incidents back to their root causes through infrastructure dependencies.
+- Determine if infrastructure is the cause of an incident, such as cross-region traffic causing latency or increased costs. 
+- Analyze and address the most relevant security misconfigurations.
+- Onboard new team members.
+- Accelerate incident MTTR and proactive governance tasks by simplifying infrastructure navigation.
+
+{{< img src="datadog_cloudcraft/cloudcraft_datadog.mp4" alt="Cloudcraft in Datadog video" video=true >}}
+
+<div class="alert alert-info">Cloudcraft in Datadog is currently only available for AWS accounts.</a></div>
+
+### Prerequisites
+
+- [Resource collection][2] must be enabled for your AWS accounts.
+- For the best experience, Datadog strongly recommends to use the AWS-managed [`SecurityAudit`][5] policy, or the more permissive [`ReadOnlyAccess`][6] policy.
+- To view security misconfigurations on the [Security findings overlay](#security-findings), [CSM][3] must be enabled.
+
+**Note**: Cloudcraft adapts to restrictive permissions by excluding inaccessible resources. For example, if you opt to not grant permission to list S3 buckets, the diagram will simply exclude those buckets. If permissions block certain resources, an alert is displayed in the user-interface(UI).
+
+## Getting started
+
+To get started using Cloudcraft, select one or more accounts, regions, and resources. The diagram automatically displays the Infrastructure diagram.
```

Comment:
I recommend including a link to the page in Datadog they're supposed to start from, since it's not clear where this action should happen.